src/include/cache.h

```diff
@@ -105,12 +105,12 @@ struct __wt_cache {
 	WT_CONDVAR *evict_cond;		/* Eviction server condition */
 	WT_SPINLOCK evict_walk_lock;	/* Eviction walk location */
 
-	u_int eviction_dirty_target;    /* Percent to allow dirty */
-	u_int eviction_dirty_trigger;	/* Percent to trigger dirty eviction */
-	u_int eviction_trigger;		/* Percent to trigger eviction */
-	u_int eviction_target;		/* Percent to end eviction */
+	double eviction_dirty_target;    /* Percent to allow dirty */
```

Comment:
Adding a comment above this block explaining why we are storing percentages in a double type would be worthwhile.